misc/Vagrantfile.tmpl

```diff
@@ -31,10 +31,10 @@ Vagrant.configure("2") do |config|
     echo './autogen.sh && ./configure --without-vdo --without-nvdimm' >> /home/vagrant/.bash_history
   SHELL
 
-  config.vm.define "bd-f28", primary: true, autostart: true do |f28|
-    # F28-specific configuration
-    f28.vm.box = "fedora/28-cloud-base"
-    f28.vm.provision :shell, inline: <<-SHELL
+  config.vm.define "bd-f29", primary: true, autostart: true do |f29|
```

Comment:
We might make this more automatic by adding some more Ruby logic to the file. A fun task for the future, I guess. :-)